src/tags/control/Taxonomy/Taxonomy.js

```diff
@@ -337,8 +338,17 @@ const Model = types
 
       path?.forEach(p => url.searchParams.append('path', p));
 
+      if (url.username && url.password) {
+        requestOptions = {
+          headers: new Headers({
+            'Authorization': `Basic ${btoa(`${url.username}:${url.password}`)}`,
+          }),
+        };
+      }
+
       try {
-        const res = yield fetch(url);
+        const res = yield fetch(url.origin + url.pathname, requestOptions);
```

Comment:
:wave: I was just skimming this PR, won't using only origin and pathname cause query params to be dropped? I think instead it'd be safer to set `url.username` and `url.password` to null after moving them onto the authorization header, then continue to yield `fetch(url)`

Reply:
yes, good catch and suggestion!